m0t1fs/linux_kernel/st/m0t1fs_sns_common_inc.sh

```diff
@@ -47,18 +47,45 @@ file=(
 	10000:10000
 	10001:10001
 	10002:10002
+	10003:10003
```

Comment:
Then, I propose to modify the following files, removing the $file, $unit_size and $file_size in them:
```
motr/st/utils/sns_repair_motr_mf.sh 
motr/st/utils/sns_repair_motr_abort.sh 
motr/st/utils/sns_repair_motr_ios_fail.sh 
```

Reply:
Agree, done.